src/index.ts

```diff
@@ -47,7 +47,7 @@ class Monika extends Command {
       char: 'c',
       description:
         'JSON configuration filename. If none is supplied, will look for default in current path.',
```

Comment:
"will look for monika.json in the current directory."

Reply:
done